internal/testplanet/uplink.go

```diff
@@ -166,8 +166,15 @@ func (uplink *Uplink) DialOverlay(destination Peer) (overlay.Client, error) {
 }
 
 // Upload data to specific satellite
-func (uplink *Uplink) Upload(ctx context.Context, satellite *satellite.Peer, bucket string, path storj.Path, data []byte) error {
+func (uplink *Uplink) Upload(ctx context.Context, satellite *satellite.Peer, redundancy *uplink.RSConfig, bucket string, path storj.Path, data []byte) error {
```

Comment:
Don't modify the existing func, create a new one. This makes other tests harder to read.